android/app/src/bt/res/values/strings.xml

```diff
@@ -6,9 +6,4 @@
   <string name="app_name_short" translatable="false">PathCheck BT</string>
   <!-- Channel label for notifications shown in App > Notifications > Path Check BT -->
   <string name="notification_channel_description">PathCheck BT notifications</string>
-
-  <string name="exposure_notification_title">Possible COVID-19 exposure</string>
-  <string name="exposure_notification_message">Someone you were near has tested positive for COVID-19. Tap for more info.</string>
-  <string name="exposure_notification_channel_name">Exposure Notification</string>
-  <string name="exposure_notification_channel_description">Exposure notification alerts.</string>
```

Comment:
These strings are required, we still need to translate them

Reply:
If those are on master those should have been updated the last time we pushed, do we know when these were added?

Reply:
They were added here https://github.com/Path-Check/gaen-mobile/pull/91
I'm not sure how's the process to get them translated, should we add them to lokalise?

Reply:
It should be happening automatically when we merge to develop, let me double-check that

Reply:
The routes on the GitHub workflow file are wrong 🤦  so changes to the strings file on android are not being picked up. Nice catch! thanks, @juancruzgs will update this

Reply:
this may be related to this card: https://trello.com/c/6AAUYzag/200-lokalize-string-bugs-in-android, @aledustet , @juancruzgs